### DISCUSSION TITLE
Simplifying the Map in navigation view

### DISCUSSION THREAD

**Discussion by Petak73** [organicmaps/organicmaps#9800] [Ideas]
Being able to remove items from the map, such as street names, buildings, building numbers, select POI icons etc. will make the navigation view much clearer, IMO.

**Comment by biodranik** [comment]
There are some other apps that can already customize anything. OM's goal is to avoid such customization by providing reasonable default parameters. What's the value of removing street names and buildings, and also POIs (often good landmarks) for navigation? Does it make sense to tune the nav map style to best fit the use-case (e.g. a car driver)?

**Reply by Petak73** [reply]
Well, in navigation view one has normally already set the destination and is focused on the road rather than looking at the map for POIs, street names etc. This seems to me to be the most common scenario and from that point of view the simpler the map view in navigation mode, the better. Fewer details means fewer distractions, which means more attention to the road ahead. Seems quite obvious to me, don't you agree?
